include/HostChecksStatus.h

```diff
@@ -31,12 +31,13 @@ class HostChecksStatus { /* Container to keep per-check status (e.g., traffic de
   u_int64_t ntp_bytes;  /* Holds the NTP bytes and is used to compute the delta of NTP bytes across consecutive check calls */
   u_int64_t p2p_bytes;  /* Holds the P2P bytes and is used to compute the delta of P2P bytes across consecutive check calls */
   u_int64_t dns_bytes;  /* Holds the DNS bytes and is used to compute the delta of DNS bytes across consecutive check calls */
+  u_int64_t my_pkt_counter;
```

Comment:
Please, give it a meaningful variable name. This is not perl and my doesn't mean anything. For the rest this is good to go.